fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/BIOCatsConversions.scala

```diff
@@ -142,9 +143,36 @@ object BIOCatsConversions {
     }
   }
 
-  class BIOCatsSync[F[+_, +_]](override val F: BIO[F]) extends BIOCatsBracket[F](F) with cats.effect.Sync[F[Throwable, ?]] {
+  trait BIOCatsParallel[F[+_, +_]] {
+    val P: cats.Parallel[F[Throwable, ?]]
+    def F: BIOParallel[F]
+  }
+
+  class BIOCatsSync[F[+_, +_]](override val F: BIO[F]) extends BIOCatsBracket[F](F) with BIOCatsParallel[F] with cats.effect.Sync[F[Throwable, ?]] {
+    self =>
     @inline override final def suspend[A](thunk: => F[Throwable, A]): F[Throwable, A] = F.flatten(F.syncThrowable(thunk))
     @inline override final def delay[A](thunk: => A): F[Throwable, A] = F.syncThrowable(thunk)
+
+    type M[A] = F[Throwable, A]
+    val P: cats.Parallel[F[Throwable, ?]] = new cats.Parallel[F[Throwable, ?]] {
```

Comment:
Have used this approach due to `cats.Parallel` having the type `F` which clashes with the classes inheriting this class

Reply:
^ See reply above, using a new class and naming the variable `F0` instead fixes this